src/com/akshay/captain/jira.main.kts

```diff
@@ -191,39 +282,159 @@ fun <E> List<E>.ifNotEmpty(function: () -> Unit): List<E> {
     return this
 }
 
-fun createReleaseVersion() {
+/**
+ * Creates a release version in the Jira
+ */
+fun createReleaseVersion(appVersion: String) {
     // create your json here
-
+    val description =
+        "https://biowink.atlassian.net/wiki/spaces/HF/pages/289570878/Android+Releases#AndroidReleases-Android$appVersion.0"
     val payload = JSONObject()
+
     try {
-        payload.put("archived", false)
-        payload.put("releaseDate", "2022-07-06")
-        payload.put("name", "New Version 1")
-        payload.put("description", "An excellent version")
-        payload.put("project", "CLUEAPP")
-        payload.put("released", false)
+        payload["archived"] = false
+        payload["releaseDate"] = "2022-07-06"
```

Comment:
question: why is this date hardcoded?

Reply:
This is just an example not the clear version for this, and this is going to removed again with change in version number